apps/ledger-live-desktop/src/renderer/screens/platform/v2/Catalog/Card/Minimum.tsx

```diff
@@ -18,8 +21,11 @@ export function MinimumCard(props: PropsRaw) {
           <Text overflow="hidden" whiteSpace="nowrap" textOverflow="ellipsis" fontSize={14}>
             {manifest.name}
           </Text>
-          {/* FIXME: Expecting ReactI18NextChildren  */}
-          <Subtitle>{manifest?.usedAt as unknown as string}</Subtitle>
+          <Subtitle>
+            {t(`common.timeAgo.${manifest.usedAt.translationKey}`, {
+              time: manifest.usedAt.diff,
+            })}
+          </Subtitle>
```

Comment:
We should probably just use `Intl.DateTimeFormat` like shown here https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-mobile/src/components/DateFormat/FormatDate.tsx#L26 or https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-desktop/src/renderer/hooks/useDateFormatter.ts#L48

Reply:
👍  This one does the trick for this PR: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat